include/IECoreHoudini/OBJ_SceneCacheNode.h

```diff
@@ -65,7 +66,10 @@ class OBJ_SceneCacheNode : public SceneCacheNode<BaseType>
 		static PRM_Name pOutTranslate;
 		static PRM_Name pOutRotate;
 		static PRM_Name pOutScale;
-		
+		OBJ_OBJECT_TYPE	 getObjectType() const
```

Comment:
I could remove this though since all it does is call the base class implementation.